.golangci.yml

```diff
@@ -9,80 +32,35 @@ output:
   # print linter name in the end of issue text, default is true
   print-linter-name: true
 
+  # make issues output unique by line, default is true
+  uniq-by-line: true
+
 linters:
   # please, do not use `enable-all`: it's deprecated and will be removed soon.
   # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
   disable-all: true
   enable:
-    - deadcode
-    - errcheck
-    - gocritic
+    # linters maintained by golang.org
     - gofmt
-    - goimports 
-    - golint 
-    - gosimple
+    - goimports
+    - golint
     - govet
-    - ineffassign
-    - interfacer
-    - lll
-    - misspell
-    - staticcheck 
-    - structcheck 
-    - typecheck 
-    - unconvert 
-    - unparam 
-    - varcheck 
+    # linters default enabled by golangci-lint .
```

Comment:
If we want to use golangci-lint, why comment out?

Reply:
linters are too much, if enable all, we should modify more codes, so just first enable some linters, we could open another PR to enable other linters

Reply:
ok, I see.